README.md

```diff
@@ -16,29 +16,45 @@ To add a new service, there is some set up involved.
 
 You will need to create some files and folders based on the name of your service, using lowercase and hypens. For example `your-service-name`.
 
+There are two options for getting started. The instructions below are the fail-safe manual way of creating a new service
+
 You’ll need to:
 
-* create a folder for your service within `app/posts/`, for example `app/posts/your-service-name/`
-* create a folder with the same name in `app/images/`, for example `app/images/your-service-name/`
-* create a `your-service-name.json` file within `app/posts/` that adds the eleventyNavigation parent name as the name of your service (copy an existing example)
-* create a Markdown file in `app/posts/`, for example `app/posts/your-service-name.md`. Copy an existing example and update any references to match your service.
-* update `eleventy.config.js` to create a new 'collection' for your service (copy one of the existing examples)
-* update `app/_layouts/_product.njk` to list your new service in the `sections` variable
+### Manually getting started
+
+- create a folder for your service within `app/posts/`, for example `app/posts/your-service-name/`
+- create a folder with the same name in `app/images/`, for example `app/images/your-service-name/`
+- create a `your-service-name.json` file within `app/posts/` that adds the eleventyNavigation parent name as the name of your service (copy an existing example)
+- create a Markdown file in `app/posts/`, for example `app/posts/your-service-name.md`. Copy an existing example and update any references to match your service.
+- update `eleventy.config.js` to create a new 'collection' for your service (copy one of the existing examples)
+- update `app/_layouts/_product.njk` to list your new service in the `sections` variable
 
 See [Divide a design history into different sections](https://x-govuk.github.io/govuk-design-history/divide-a-design-history-into-sections/).
 
+### Scripted getting started.
+
+An alternative method is to run a script that will create a service and first post for you. It assumes you are working on a linux based system with access to bash
+
+```
+./make-design-history.sh "your service name" "Screening|Vaccinations|Healthchecks"
```

Comment:
Instead of "Healthchecks" could we use "Personalised prevention services"? cc @ralph-hawkins